core/jazz_services/components/crud/getSearchList.js

```diff
@@ -0,0 +1,159 @@
+// =========================================================================
```

Comment:
getSearchList.js is practically the same as getList.js except probably for one line - reuse the code and make it DRY?

Reply:
Addressed with conditions.